Sources/ClangImporter/ClangImporter.swift

```diff
@@ -345,7 +314,7 @@ public class ClangImporter: Pass {
       let name = Identifier(name: clang_getCursorSpelling(child).asSwift())
       if context.global(named: name) != nil { return CXChildVisit_Continue }
 
-      let range = SourceRange(clangRange: clang_getCursorExtent(child))
+      let range = self.self.makeRange(clang_getCursorExtent(child))
```

Comment:
self.self.?

Reply:
oops, accidentally applied a fix-it after I'd fixed it